trade-finance-manager-api/src/constants/tasks.js

```diff
@@ -157,7 +157,7 @@ const MIA = {
 
 const STATUS = {
   TO_DO: 'To do',
-  IN_PROGRESS: 'In progress',
+  IN_PROGRESS_BY_UKEF: 'In progress',
```

Comment:
Not a fan of this one as all task statuses will be "by ukef" if you see what I mean

Reply:
makes sense. I've updated this now